### PR TITLE
fix: Fixed failing tests in `tests/utils/test_add_new_model_like.py`

### PR DESCRIPTION
**Pull request by Sai-Suraj-27**
# What does this PR do?
Fixed, the following failing tests in `tests/utils/test_add_new_model_like.py`
```
tests/utils/test_add_new_model_like.py::TestAddNewModelLike::test_get_model_files
tests/utils/test_add_new_model_like.py::TestAddNewModelLike::test_get_model_files_only_pt
tests/utils/test_add_new_model_like.py::TestAddNewModelLike::test_get_model_files_tf_and_flax
tests/utils/test_add_new_model_like.py::TestAddNewModelLike::test_retrieve_info_for_model_with_vit
tests/utils/test_add_new_model_like.py::TestAddNewModelLike::test_retrieve_info_for_model_with_wav2vec2
tests/utils/test_add_new_model_like.py::TestAddNewModelLike::test_retrieve_model_classes
```
Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#create-a-pull-request),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?
@amyeroberts @LysandreJik

### CODE REVIEW

File: tests/utils/test_add_new_model_like.py [hunk 1]
```diff
@@ -836,7 +843,7 @@ def test_retrieve_info_for_model_with_wav2vec2(self):
         ]
         expected_model_classes = {
             "pt": set(wav2vec2_classes),
-            "tf": {f"TF{m}" for m in wav2vec2_classes[:1]},
+            "tf": {f"TF{m}" for m in wav2vec2_classes[:1] + ["Wav2Vec2ForSequenceClassification"]},
```
Comment (amyeroberts) [inline tests/utils/test_add_new_model_like.py:846]: I think it'd be better to still use the `wav2vec2_classes` object so something like: 

```suggestion
            "tf": {f"TF{m}" for m in [wav2vec2_classes[0], wav2vec2_classes[-2]]},
```

**Review by amyeroberts — APPROVED** [review]
Thanks for updating! 

Just one small nit

File: tests/utils/test_add_new_model_like.py (hunk 1 above)
Comment (Sai-Suraj-27) [reply tests/utils/test_add_new_model_like.py]: Done 👍🏻

Comment (amyeroberts) [reply tests/utils/test_add_new_model_like.py]: Thanks!

**Pull request merged**